index.html.slim

```diff
@@ -37,9 +33,13 @@ description: The simple, modern & productive way to build apps and infrastructur
           img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_380x280_devstudio10_16jun2016.png") alt="DevStudio 10 includes OpenShift integration. Join RHD to download it now.")
           img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x500_devstudio10_16jun2016.png") alt="DevStudio 10 includes OpenShift integration. Join RHD to download it now. ")
       .large-24.slide
-        a( href="http://developers.redhat.com/products/openjdk/overview/")
-          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_tablet_480x500_openjdk_16jun2016.png") alt="OpenJDK is now available for Windows. Join RHD now to download.")
-          img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x400_openjdk_16jun2016.png") alt="OpenJDK is now available for Windows. Join RHD now to download.")
+        a( href="http://developers.redhat.com/promotions/microservices-for-java-developers/")
+          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_tablet_480x500_microservices_09jun2016.jpg") alt="Get a hands-on introduction to frameworks and container.")
+          img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x400_microservices_09jun2016.jpg") alt="Get a hands-on introduction to frameworks and container.")
+      .large-24.slide
+        a( href="http://developers.redhat.com/events/devnation/2016/")
+          img.mobile(src="http://static.jboss.org/images/hero/hero_tablet_480x500_devnationrecap_16jun2016.jpg" alt="Get a recap of the DevNation 2016 event and access to the on demand content.")
+          img.desktop(src="http://static.jboss.org/images/hero/hero_desktop_480x500_devnationrecap_16jun2016.jpg" alt="Get a recap of the DevNation 2016 event and access to the on demand content..")
```

Comment:
Should it use this image instead? 

http://static.jboss.org/images/hero/hero_desktop_1175x400_devnationrecap_16jun2016.jpg